internal/pkg/controllers/execLog.go

```diff
@@ -0,0 +1,218 @@
+package controllers
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	options "github.com/breathbath/go_utils/v2/pkg/config"
+	"github.com/breathbath/go_utils/v2/pkg/fs"
+	"github.com/cloudradar-monitoring/rportcli/internal/pkg/config"
+	"github.com/cloudradar-monitoring/rportcli/internal/pkg/models"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	overwriteExistingLogFileMsg    = "Overwrite existing execution log file (y/n): "
+	WillBeExecutedWithClientIDsMsg = "Your task will be executed on the following clients:"
```

Comment:
If I use `--readexeclog log.yaml` and the log does not contain a previously failed execution I get 
```
Your task will be executed on the following clients:
FATAL: Failed to create script on multiple clients, code: , details: Client with id="" not found.
```
I expect something like "Nothing to do. Logfile does not contain clients with failed execution."

Reply:
should caught this one. but done with test.